### PR TITLE
Adding support for Slack

### PR DESCRIPTION
**Pull request by kzapolski**

### CODE REVIEW

**Review by alertedsnake — APPROVED** [review]
Looks great!

File: rssalertbot [hunk 1]
```diff
@@ -272,6 +287,36 @@ class Feed:
         log.warning(f"({self.feed}) {entry.published}: {entry.title}")
         log.debug(f"{entry.description}")
 
+    def alert_slack(self, entry):
+        """Sends alert slack"""
+
+        if not self.enable_slack:
+            return
+
+        attachments = attachments = [
```
Comment (alertedsnake) [inline rssalertbot]: attachments * 2?

File: rssalertbot [hunk 2]
```diff
@@ -348,13 +380,13 @@ def guess_color(matchstring):
 
     for key in KEYS_GREEN:
         if key in matchstring:
-            return 'green'
+            return {"hipchat_color": "green", "slack_color": "#3aa459"}
```
Comment (alertedsnake) [inline rssalertbot]: I'd be tempted to just use the slack default color names here, the `danger` and `warning` names.

File: rssalertbot [hunk 3]
```diff
@@ -272,6 +293,62 @@ class Feed:
...
+                ],
+                "color": color
+            }
+        ]
+
+        try:
+            sc = SlackClient(self.slack_token)
+            sc.api_call(
+                "chat.postMessage",
+                channel=self.slack_channel,
```
Comment (alertedsnake) [inline rssalertbot:346]: For me, I had to add 'username' here to get something relevant to show up in the list, I'd like to see what this looks like for you as-is.  But I guess it's just a minor thing.

Actually, attaching an image to the PR would be cool :)

File: rssalertbot (hunk 2 above)
Comment (kzapolski) [reply rssalertbot]: that's good suggestion - added good / warning / danger

File: rssalertbot (hunk 3 above)
Comment (kzapolski) [reply rssalertbot:346]: we can make icon_emoji=':information_source:' configurable

Comment (kzapolski) [reply rssalertbot:346]: image

Comment (alertedsnake) [reply rssalertbot:346]: Ooh that looks really good!  But yeah, set `username` in the call it'll replace the word 'bot' there with something, like "rssalertbot", so it's clear what it is.

Comment (kzapolski) [reply rssalertbot:346]: image

**Comment by kzapolski** [comment]
image

**Pull request merged**